doubly_linked_list/doubly_linked_list.py

```diff
@@ -45,19 +45,53 @@ def __len__(self):
 
   def add_to_head(self, value):
     if self.head is None:
-      new_node = ListNode(value)
+      # allocate the Node and put in teh data
+      new_node = ListNode(value = value)
+
+      new_node.next = self.head
+      new_node.prev = None
+      if self.head is not None:
+        self.head.prev = new_node
```

Comment:
are these tests passing? At first glance I notice that you aren't incrementing or decrementing when you add a node or remove a node, but perhaps you have changed that and the updated code hasn't been pushed up yet. As for this method specifically, I am a little unsure if the tests would pass because I feel like you need to have a variable to hold onto the value that existed before you added to the head (if there was one). Similar to what you did when you added to the tail.